Packages/mormot_base.lpk

```diff
@@ -9,17 +9,11 @@
       <Version Value="11"/>
       <PathDelim Value="\"/>
       <SearchPaths>
-        <IncludeFiles Value="..;..\SQLite3"/>
+        <IncludeFiles Value="$PkgDir(zcore)\..\..\src;..;..\SQLite3"/>
```

Comment:
We need this path because `SynDBZeos` uses `Zeos.inc`. It's not enough just add `zcomponent` into `mormot_base`.